components/logins/src/migrate_sqlcipher_db.rs

```diff
@@ -586,47 +586,32 @@ mod tests {
         db
     }
 
-    fn create_old_db(db_path: impl AsRef<Path>, salt: Option<&str>) {
+    fn create_old_db_with_test_data(db_path: impl AsRef<Path>, salt: Option<&str>, inserts: &str) {
         let mut db = open_old_db(db_path, salt);
         let tx = db.transaction().unwrap();
         schema::init(&tx).unwrap();
 
-        // Manually migrate back to schema v4 and insert some data
-        // These all need to be executed as separate statements or
-        // sqlite will not execute them
-        const RENAME_LOCAL_USERNAME: &str = "
-            ALTER TABLE loginsL ADD COLUMN username
-        ";
-
-        const RENAME_LOCAL_PASSWORD: &str = "
-            ALTER TABLE loginsL ADD COLUMN password
-        ";
-
-        const RENAME_MIRROR_USERNAME: &str = "
-            ALTER TABLE loginsM ADD COLUMN username
-        ";
-
-        const RENAME_MIRROR_PASSWORD: &str = "
-            ALTER TABLE loginsM ADD COLUMN password
-        ";
-
-        const RENAME_LOCAL_HOSTNAME: &str = "
-            ALTER TABLE loginsL RENAME origin TO hostname
-        ";
-
-        const RENAME_LOCAL_SUBMIT_URL: &str = "
-            ALTER TABLE loginsL RENAME formActionOrigin TO formSubmitURL
-        ";
-
-        const RENAME_MIRROR_HOSTNAME: &str = "
-            ALTER TABLE loginsM RENAME origin TO hostname
-        ";
-
-        const RENAME_MIRROR_SUBMIT_URL: &str = "
-            ALTER TABLE loginsM RENAME formActionOrigin TO formSubmitURL
+        // Note that we still abuse our current schema for this. As part of the migration away
+        // from sqlcipher we renamed some columns, which we need to rename back.
+        // (The alternative would be to clone the entire schema from the last sqlcipher version,
+        // which isn't really any better than this, so meh.)
+        const RENAMES: &str = "
```

Comment:
Nice! For whatever reason I couldn't get this to work all in one string for me (even with the semis)